wlc/test_base.py

```diff
@@ -62,7 +62,12 @@ def format_body(self, body):
             return ""
         body = body.decode()
         body = (
-            body.replace(": ", "=").replace("{", "").replace("}", "").replace('"', "")
+            body.replace(": ", "=")
+            .replace("{", "")
+            .replace("}", "")
+            .replace('"', "")
+            .replace("://", "-::")
+            .replace("/", ":")
         )
 
         return body
```

Comment:
Might be worth adding:
```
body.replace(", ", "--")
```
Here to prevent spaces within the filename when multiple kwargs are parsed to the request.
Helps to give test file names cleaner.

Reply:
Sounds like a good idea. Let me update and check once.

Reply:
Works fine.